DemoApp/DemoApp/Views/PopupSamples/SampleCustomPopup.xaml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<popup:PopupPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
Popup should be from modus components please check the namespace

Reply:
Done